driver-core/src/test/java/com/datastax/driver/core/ExtendedPeerCheckDisabledTest.java

```diff
@@ -0,0 +1,36 @@
+/*
+ *      Copyright (C) 2012-2015 DataStax Inc.
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ */
+package com.datastax.driver.core;
+
+import org.testng.annotations.Test;
+
+public class ExtendedPeerCheckDisabledTest {
+
+    /**
+     * Validates that if the com.datastax.driver.EXTENDED_PEER_CHECK system property is set to false that a peer
+     * with null values for host_id, data_center, rack, tokens is not ignored.
+     *
+     * @test_category host:metadata
+     * @jira_ticket JAVA-852
+     * @since 2.1.10
+     */
+    @Test(groups = "isolated", dataProvider = "disallowedNullColumnsInPeerData", dataProviderClass = ControlConnectionTest.class)
```

Comment:
Do we really need isolated profile because of system properties?
I thought a try-finally block could suffice, e,g, [here](https://github.com/datastax/java-driver/blob/47419b8be04758d22d77534402c4d4c613a41694/driver-core/src/test/java/com/datastax/driver/core/SSLEncryptionTest.java#L118-L131).